omnibus/config/software/installer.rb

```diff
@@ -47,6 +47,7 @@
     # Packages
     mkdir "/opt/datadog-packages"
     copy 'bin/installer', "#{install_dir}/bin/"
+    copy '#{files_path}/installer/Readme.md', "#{install_dir}/Readme.md"
```

Comment:
You'll need to replace single quotes with double quotes so that the value can be substituted